app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -326,6 +321,11 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
             null -> layoutTypeMenuItem?.isVisible = false
         }
 
+        viewModel.duckChatVisibility.observe(this) {
+            val duckChatMenuItem = menu.findItem(R.id.duckChat)
+            duckChatMenuItem?.isVisible = it
+        }
```

Comment:
`onCreateOptionsMenu` can be called multiple times (Like if you rotate), so might cause issues. Rather observe in `onCreate`.

I tested the following and it worked, just add a new variable:
```
private var duckChatMenuItem: MenuItem? = null
```

Then in `configureObservers` add:

```
viewModel.duckChatVisibility.observe(this) {
    duckChatMenuItem?.isVisible = it
}
```

And in `onCreateOptionsMenu` do this instead:
```
duckChatMenuItem = menu.findItem(R.id.duckChat)
duckChatMenuItem?.isVisible = viewModel.duckChatVisibility.value ?: false
```